taverna-scufl2-cwl/src/main/java/org/apache/taverna/scufl2/cwl/CWLParser.java

```diff
@@ -22,16 +22,10 @@
 import java.util.Set;
 import java.util.HashSet;
 
-import org.apache.taverna.scufl2.api.core.Workflow;
-import org.apache.taverna.scufl2.api.core.Processor;
-
-import org.apache.taverna.scufl2.api.port.InputWorkflowPort;
-import org.apache.taverna.scufl2.api.port.OutputWorkflowPort;
-import org.apache.taverna.scufl2.api.port.InputProcessorPort;
-import org.apache.taverna.scufl2.api.port.OutputProcessorPort;
-
 import com.fasterxml.jackson.databind.JsonNode;
 
+import org.apache.taverna.scufl2.cwl.components.*;
```

Comment:
No, don't use `.*` imports, that can be very fragile and confusing, in particular once you get two `*` imports that have the same class name.